SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where-Object { $_.productType -match "NSXMgr" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,apiPasswdMaxFailAttempts,apiPasswdMaxFailIntervalInSec,apiPasswdUnlockIntervalInSec,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare NSX Edge servers section"
+	$htmlOutput = $htmlOutput + '<H3>NSX Edge Password Policy</h3><table class="myTable">'
+	$tmpHtmlOutput = $ppmEnvironmentalDetails | where-Object { $_.productType -match "NSXEdge" } | select-object -Property FQDN -ExpandProperty productAttributes | select-object FQDN,passwdMinimumLength,cliPasswdMaxFailAttempts,cliPasswdMaxFailIntervalInSec | sort-object -Property FQDN | ConvertTo-Html -Fragment -As Table
+	$htmlOutput += $tmpHtmlOutput
+	#prepare Workspace ONE Access servers section"
+	if($ppmVariables.skipWSA -eq $false){
+		$htmlOutput = $htmlOutput + '<H3>Workspace ONE Access Password Policy</H3><p><font color=#6495ED>WSA FQDN: ' + $ppmVariables.wsaFqdn + '</font></p><table class="myTable">'
+		$tmpHtmlOutput = $ppmEnvironmentalDetails | where-Object { $_.productType -match "WSA" } | select-object -ExpandProperty productAttributes | select-object -property * -ExcludeProperty FQDN,productType,driftAlarm,driftMessage | ConvertTo-Html -Fragment -As Table
+		$htmlOutput += $tmpHtmlOutput
+	}
+	else{
+		$htmlOutput = $htmlOutput + '<H3>Workspace ONE Access Password Policy</h3><p><br><font color=>Workspace One Access Skipped, No environmental details provided</p>'
+	}
+	
+	$htmlOutput = $htmlOutput.replace("<table>", "")
+	$htmlOutput = $htmlOutput.replace("passwd", "passwd ")
+	$htmlOutput = $htmlOutput.replace("&gt;", ">")
+	$htmlOutput = $htmlOutput.replace("&lt;", "<")
+	$htmlOutput = $htmlOutput.replace("<colgroup><col/><col/><col/></colgroup> ","")
+	$htmlOutput = $htmlOutput + "</body></html>"
+	$htmlOutput | out-File $ppmVariables.outputFilePath
+	Write-LogMessage -type INFO -Message "$($ppmVariables.outputFilePath) report generated" -Colour Yellow
+}
+
+################################################################################
+####   Print current environment password policies settings as JSON file	####
+################################################################################
+Function ExportReportJSON {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$ppmVariables,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues,
+        [Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmEnvironmentalDetails,
+		[Parameter (mandatory = $false)] [Switch]$doComparison = $false,
+		[Parameter (mandatory = $false)] [Switch]$onlyDrift = $false
+	)
```

Comment:
Add line space after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175